service/src/main/java/com/zebrunner/reporting/service/impl/StorageServiceImpl.java

```diff
@@ -110,16 +129,43 @@ private String getKeyPrefix(BinaryObject.Type type) {
         String prefix = "";
         switch (type) {
             case ORG_ASSET:
-                prefix = "assets/org";
+                prefix = ORG_ASSET_PREFIX;
                 break;
             case USER_ASSET:
-                prefix = "assets/user";
+                prefix = USER_ASSET_PREFIX;
                 break;
             case APP_PACKAGE:
-                prefix = "artifacts/applications";
+                prefix = APP_PACKAGE_PREFIX;
```

Comment:
Please try to eliminate such kind of enums. Or at least return the prefix immediately from a case section without redundant variable. This is valid for the entire pull request.

Reply:
Sorry, I love enums

Reply:
I think that wrapping binary file types and ampq messaging properties into multiple enums at least make code less readable: not having real values in sight results in extra time to look for them. Maybe you still consider refactoring?

Reply:
There are many places where this value is using